### DISCUSSION TITLE
custom chroot?

### DISCUSSION THREAD

**Discussion by USMP-lancer** [dnschneid/crouton#4548] [Q&A]
would it be possible to setup a chroot with an iso, such as an android chroot?

**Comment by ghost** [comment]
Chroots just hold a copy of the system's files, so you could, but it wouldn't be easy. I don't even know if Crouton would be the right program for that. You can start chroots by use the `chroot` command. I suggest, maybe, that you mount the ISO and try to copy the files... 

I also think that chroots must be other Linux systems.

**Reply by USMP-lancer** [reply]
android has a modified version of the linux kernal, so its possible it might work

**Reply by USMP-lancer** [reply]
the only problem i see is i cannot get an iso for arm64, best thing is the gsi from google, which probable wont work

**Reply by ghost** [reply]
I *should* be able to get any archive, that you can mount, extract, etc. , to work... I don't know how well though.

**Reply by USMP-lancer** [reply]
the only reason i want to set this up is i want to emulate android on arm64, if i can do that somehow, ill likely switch to installing actual linux on my machine
